lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -43,7 +43,8 @@
     <p>
       <input type="button" name="calculate-grades-csv" value="${_("Generate Grade Report")}" data-endpoint="${ section_data['calculate_grades_csv_url'] }"/>
       <input type="button" name="get-student-responses" value="${_("Get Student Responses Report")}" data-endpoint="${ section_data['get_student_responses_url'] }"/>
-      <input type="button" name="ora2-response-btn" value="${_("Download Ora2 Responses")}" data-endpoint="${ section_data['get_ora2_responses_url'] }"/>
+      <input type="button" name="ora2-response-btn" value="${_("Download Ora2 Responses (no email addresses)")}" data-endpoint="${ section_data['get_ora2_responses_url'] }"/>
+      <input type="button" name="ora2-response-btn-email" value="${_("Download Ora2 Responses (including email addresses)")}" data-endpoint="${ section_data['get_ora2_responses_url_email'] }"/>
```

Comment:
as far as naming convention, would it make more sense to use

get_ora2_email_responses_url  instead?

This would be more consistent with how other endpoints are named